### PR TITLE
Add `junifer` to `conda-forge`

### PR DESCRIPTION
**Pull request by synchon**
Make `junifer` available via `conda` by adding recipe to [conda-forge](https://conda-forge.org) channel. This will enable users to install `junifer` via:
```shell
conda install -c conda-forge junifer
```

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -48,6 +48,12 @@ Use ``pip`` to install julearn from `PyPI <https://pypi.org>`_, like so:
 
     pip install junifer
 
+You can also install via ``conda``, like so:
```
Comment (LeSasse) [inline docs/installation.rst:51]: Same as above: Would it make sense to let users know that there may be some issues with git-annex/datalad on macs?

**Review by LeSasse — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ Use `pip` to install from PyPI like so:
 pip install junifer
 ```
 
+You can also install via `conda`, like so:
+
+```
```
[the remark this answers is not in the document]
Comment (synchon) [reply README.md:48]: Yeah of course, I'll make the changes.

[the remark this answers is not in the document]
Comment (fraimondo) [reply README.md:48]: which issues?

I use to have issues, but not recently.

[the remark this answers is not in the document]
Comment (synchon) [reply README.md:48]: I think it's best to keep the information in the docs and skip it in the README for brevity. What do you think?

[the remark this answers is not in the document]
Comment (synchon) [reply README.md:48]: > which issues?
> 
> I use to have issues, but not recently.

Installation failure on Apple Silicon and also git-annex only available for linux on conda-forge. I already had a chat with the datalad folks and looks like they will come up with a fix.

[the remark this answers is not in the document]
Comment (LeSasse) [reply README.md:48]: > I think it's best to keep the information in the docs and skip it in the README for brevity. What do you think @LeSasse ?

Yes makes sense to me, I think thats where most people will look at it anyhow.

**Pull request merged**